persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/MushroomHunter.java

```diff
@@ -13,12 +15,8 @@
     @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Long id;
 
-    @NotNull
-    @Column(nullable=false,unique=true)
     private String firstName;
 
-    @NotNull

```

Comment:
My opinion is that @NotNull could stay, just unique was problematic. You probably want a hunter to have a name set